assets/src/scripts/__tests__/components/MoreInfoModal.spec.tsx

```diff
@@ -90,12 +81,11 @@ it("shows 'No synonyms' if fetch fails", async () => {
   );
   const user = userEvent.setup();
   render(<MoreInfoModal {...props} />);
-  user.click(screen.getByRole("button", { name: /more info/i }));
 
-  // expect(await screen.findByText(/loading synonyms/i)).toBeInTheDocument();
-  await waitFor(() => {
-    expect(screen.getByText(/no synonyms/i)).toBeInTheDocument();
+  await act(async () => {
+    await user.click(screen.getByRole("button", { name: /more info/i }));
```

Comment:
Based on the other tests can we get rid of the `await act()` wrapper function here?